src/nrnpython/CMakeLists.txt

```diff
@@ -15,10 +15,23 @@ install(TARGETS rxdmath DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
 # =============================================================================
 set(nrnpython_lib_list)
 
+# nrnpython sources (only if ${NRN_ENABLE_PYTHON_DYNAMIC} is OFF}
+set(NRNPYTHON_FILES_LIST
```

Comment:
Isn't the comment lying? We use `NRNPYTHON_FILES_LIST` even if `NRN_ENABLE_PYTHON_DYNAMIC` is ON.
Maybe we could define the `add_library(nrnpython OBJECT )` here instead and use the object library in both situations

Reply:
I will change the comment. The second idea is really good except we use different python include files when building in dynamic. So we cannot "objectify" it for once and all.

Reply:
Oh, that's true! :( 👍